apps/jest.config.js

```diff
@@ -25,6 +25,21 @@ Object.entries(APPLICATION_ALIASES).forEach(([alias, localPath]) => {
   jestAliases[`${alias}/(.*)`] = `${localPath}/$1`;
 });
 
+/**
+ * List of node_modules to transform with babel-jest (default is not to transform)
```

Comment:
default is to transform, right?

Reply:
hoo boy, we might have a double negation at play here, and its ambiguous which layer I'm speaking of 😆 

From the perspective of "node modules", and the `nodeModulesToTransform` property this is a comment on, we've configured our default to be "do not transform" (i.e. we don't apply babel to node_modules/ packages unless they're explicitly in this new list).

On the other hand, this property is used to configure the jest property `transformIgnorePatterns`, where the default is "transform unless listed".

Reply:
I just removed the `default is not to transform` language, since it didn't really add anything and causes confusion.